lib/spack/spack/cmd/help.py

```diff
@@ -47,6 +47,7 @@
       @r{-variant} or @r{~variant}          disable <variant>
```

Comment:
Maybe change the order here while you're at it?


```suggestion
      @r{~variant} or @r{-variant}          disable <variant>
```